config/install/paragraphs.paragraphs_type.oe_accordion_item.yml

```diff
@@ -0,0 +1,8 @@
+langcode: en
+status: true
+dependencies: {  }
+id: oe_accordion_item
+label: 'Accordion item'
+icon_uuid: null
+description: "Accordions are a collection of collapsible items, each of which is composed of the following fields:\r\nAn icon\r\nA title\r\nA long text"
```

Comment:
Change description into something that actually describes the "Accordion item", something like: 
> Single accordion item, to be used exclusively in conjunction with the "Accordion" paragraph type.